packages/@okta/vuepress-site/docs/guides/configure-native-sso/main/index.md

```diff
@@ -0,0 +1,440 @@
+    "scope": "openid device_sso",
+    "id_token": "eyJraWQiOiJZQlRCZUNBOThRa3pSUGdBQzZSS3BaM0J5ZlRwTXRMMHRITkp1LTQ4VVVFIiwiYWxnIjoiUlMyNTYifQ.eyJzdWIiOiIwMHV5MjcyZ2RkTFVER2pGYTB4NiIsInZlciI6MSwiaXNzIjoiaHR0cHM6Ly9jaHJpc3RpbmVjaG95LW9wMi5va3RhcHJldmlldy5jb20vb2F1dGgyL2RlZmF1bHQiLCJhdWQiOiIwb2ExamdxbHFtcmJPTXNHcDB4NyIsImlhdCI6MTYyODAzMjkzMCwiZXhwIjoxNjI4MDM2NTMwLCJqdGkiOiJJRC55VDFjVFZVdkV0STZmenBETzRYZkVadk5xdjVlVUhwT0IwUC1mMmVPcERBIiwiYW1yIjpbInB3ZCJdLCJpZHAiOiIwMG95MjcyY3NhR0NNamRPVjB4NiIsInNpZCI6IjEwMm9SZ0VXeDVsUlRXSGlsRW9HZmVkNHciLCJhdXRoX3RpbWUiOjE2MjgwMjcxMzcsImF0X2hhc2giOiJGVy1ZclFoXzhQOThmQ1NqUEQxQW1BIiwiZHNfaGFzaCI6IjN0Z3AtUFVuX213MUtkZHdGay1TYUEifQ.lHclmmSNV1ns2GGJCxEBRZpMCp_xAePqAZQvnwJ5207Gr5ohbmPkXYmgI6zA9STTyfPgiMVotJG0Iir2Ir9Ke9PBvcd6fvUTZPZTzg3McMBfRoaF2ORYO-1qoUIIEGUDtYqiUosmc_OR27U-AS6TZfnMo4Ed7KIAIHlPgzJDlAFCCa1vx-ImUCt0czP8eehEqSDE1nB1oPj3uvGpjsJxj4wwFvGXqGemHpWDbvTDvwNv5dGm05q6LNYZjJACyegtIagvKgFv5mZquR-tlXIS6SEynFNsNP_YEf-gNZ36I1y-wk6tjmTgYijcZPcvVl9PvE4giokjXVjPsjP3WGQWgA",
+    "device_secret": "kyLmBvfjMPnMGHtBhW9nKz0tLgJc5xbLay1ujkvOQIk="
+}
+```
+
+## Validate the device secret
+
+Occasionally you may want to verify that the device secret is still valid by using the `/introspect` endpoint.
+
+**Example introspect request**
+
+```bash
+  curl --request POST \
+  --url https://${yourOktaDomain}/oauth2/v1/introspect \
+  --header 'Accept: application/json' \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data-urlencode 'client_id={client #1 id}' \
+  --data-urlencode 'token={device_secret}' \
+  --data-urlencode 'token_type_hint=device_secret'
+```
+
+**Example introspect response**
+
+```json
+{
+    "active": true,
+    "sid": "102oRgEWx5lRTWHilEoGfed4w",
+    "token_type": "urn:x-oath:params:oauth:token-type:device-secret"
+}
+```
+
+The `/introspect` endpoint returns the `sid` that it's tied to. The same value is present in the ID token. By doing this, you can correlate and identify the ID tokens that are minted with the same device secret.
+
+## Revoke the device secret to end a desktop session
+
+Sometimes you have to end a user's desktop session. When you do that, you are signing the user out of one app as well as all of the other registered apps. To end a desktop session, you must revoke the device secret. The revoke request signs the user out from all of the apps that are a part of the Native SSO flow.
+
+**Example request**
+
+```bash
+curl --location --request POST \
+--url https://${yourOktaDomain}/oauth2/{authorizationServerId}/v1/revoke \
+--header 'Accept: application/json' \
+--header 'Content-Type: application/x-www-form-urlencoded' \
+--data-urlencode 'token={device_secret}' \
+--data-urlencode 'client_id={client #1 id}' \
+```
+
+**Example response**
+
+```
+HTTP/1.1 200 OK
+Content-Type: application/json
```

Comment:
If this is what we display when there's no response body, then we must be consistent and provide this response on the other requests in this guide (and in https://github.com/okta/okta-developer-docs/pull/2397).

Also, I doubt that there's a `Content-Type` header present on the response if there is no content body. That'd be going against the `http` protocol. Please check again, I think what you'd want is just:
```suggestion
HTTP/1.1 200 OK
```

Reply:
Fixed